packages/immutable-rootfs/30cos-immutable-rootfs/module-setup.sh

```diff
@@ -29,7 +29,7 @@ install() {
     # Include utilities required for cos-setup services,
     # probably a devoted cos-setup dracut module makes sense
     inst_multiple -o \
-        partprobe sync udevadm lsblk sgdisk parted mkfs.ext2 mkfs.ext3 mkfs.ext4 mkfs.vfat mkfs.fat mkfs.xfs blkid e2fsck resize2fs mount xfs_growfs umount
+        /usr/lib/systemd/systemd-fsck partprobe sync udevadm lsblk sgdisk parted mkfs.ext2 mkfs.ext3 mkfs.ext4 mkfs.vfat mkfs.fat mkfs.xfs blkid e2fsck resize2fs mount xfs_growfs umount
```

Comment:
this *should* be already included - BUT - I feel more safe being explicit here. I'm not sure if there is any reason to NOT do it.

Reply:
I don't think there is any reason not to do it. However this is part of the `00systemd` module here https://github.com/dracutdevs/dracut/blob/1befc6416743a527824f0f2cc25471e86a6f8a79/modules.d/00systemd/module-setup.sh#L33-L41 so from a dracut perspective I guess the proper way of enforcing it would be to require `systemd` module in `depends` function. I doubt we explicitly require systemd module anywhere, assuming it is already there, which could not be the case.

So, I'd suggest adding `systemd` in depends, then keep this `systemd-fsck`inclusion here is good to explicitly tell we make use of systemd-fsck. I am wondering if shouldn't we use the `${systemdutildir}` variable instead of the absolute path, just incase there are divergences on distros or the installation path of systemd evolves.